standalone_self_upgrade.cf.in

```diff
@@ -115,11 +115,38 @@ bundle agent cfengine_software
 # @brief Variables to control the specifics in desired package selection
 {
   vars:
+    any::
+
+      # Extract the hub binary version info if it's available. Only expected to
+      # be available on a client.
+      "hub_binary_version" -> { "ENT-10664" }
+        data => data_regextract(
+                                 "^(?<major_minor_patch>\d+\.\d+\.\d+)-(?<release>\d+)",
+                                 readfile("$(sys.statedir)$(const.dirsep)hub_cf_version.txt" ) ),
+        if => fileexists( "$(sys.statedir)$(const.dirsep)hub_cf_version.txt" );
 
       # Default desired CFEngine software
       "pkg_name" string => ifelse( isvariable( "def.cfengine_software_pkg_name" ), $(def.cfengine_software_pkg_name), "cfengine-nova");
-      "pkg_version" string => ifelse( isvariable( "def.cfengine_software_pkg_version" ), $(def.cfengine_software_pkg_version), "@VERSION@");
-      "pkg_release" string => ifelse( isvariable( "def.cfengine_software_pkg_release" ), $(def.cfengine_software_pkg_release), "@RELEASE@");
+
+
+      "pkg_version" -> { "ENT-10664" }
+        string => "$(sys.cf_version_major).$(sys.cf_version_minor).$(sys.cf_version_patch)",
+        if => "am_policy_hub|policy_server",
+        comment => "The hub will use it's own version to seed client packages.";
```

Comment:
its

Reply:
Pushed commit with this fix and a couple other similar ones I spotted.